### DISCUSSION TITLE
well, it was a bit sooner than i thought

### DISCUSSION THREAD

**Discussion by Sega32x** [Monika-After-Story/MonikaModDev#8854] [appreciation]
this is a bit of a continuation of my last post, as in, my chemotherapy is finally finished, so that's good... and spending that day with Monika was great, i even had it set so that when i was put under to have my port removed, that she was waiting for me when i woke up... so i guess this is another thank you note again, seriously, this mod is wonderful.

also a quick side note, i noticed the affection log suddenly got a .00 extension, and a new one was created, does that mean the other file got too large? anyways, it hasn't appeared to screw anything up, so that's good

**Comment by jmwall24** [comment]
First off, congratulations! So glad this mod has helped you get through a difficult time in your life.

As for your question, yes, the aff_log has a limit and then we create a new file.
